### PR TITLE
Fix broken politico.com ruleset and add tests; fixes #9000

### LINKED ISSUE #9000: politico.com ruleset broken
Politico.com URLs are being rewritten incorrectly.

For example:

    http://www.politico.com/agenda/story/2017/03/national-obesity-tracking-experiemnt-000335

Is rewritten to:

    https://secure.politico.com/agenda/story/2017/03/national-obesity-tracking-experiemnt-000335

### PR DESCRIPTION
**Pull request by deutrino**
This fixes #9000

### CODE REVIEW

**Comment by J0WI** [comment]
see also #9361

File: src/chrome/content/rules/Politico.xml [hunk 1]
```diff
@@ -61,16 +63,41 @@
...
+
+	<exclusion pattern="^http://(?:www\.)?politico\.com/agenda" />
+		<test url="http://www.politico.com/agenda/" />
+		<test url="http://www.politico.com/agenda/story/2015/05/what-does-hillary-stand-for-check-her-staff-twitter-000057" />
+
+	<exclusion pattern="^http://(?:www\.)?politico\.com/magazine" />
+		<test url="http://www.politico.com/magazine/" />
+		<test url="http://www.politico.com/magazine/story/2014/01/waterboarding-cia-lawyer-john-rizzo-torture-101758" />
+
+	<exclusion pattern="^http://(?:www\.)?politico\.com/[0-9]+-election/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Politico.xml]: Please avoid non-capturing groups.

Comment (gordon-morehouse) [inline src/chrome/content/rules/Politico.xml]: Removed.

File: src/chrome/content/rules/Politico.xml [hunk 2]
```diff
@@ -61,16 +63,41 @@
 
 		<test url="http://static.politico.com/cf/05/ee684a274496b04fa20ba2978da1/politico.png" /><!--	1064 -->
 
-
 	<securecookie host="^\." name="^(?:_ga|optimizely)" />
 	<securecookie host="^\w" name="." />
 
+	<exclusion pattern="^http://tk.politico.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Politico.xml]: @deutrino would be great if you could fix this.

**Pull request merged**